Test.cs

```diff
@@ -12,7 +14,12 @@ public static long calculate(long number)
                 throw new Exception("Number must be non-negative.");
             }
 
-            return number == 0 ? 1 : number * Calculate(number + 1);
+            return number == 0 ? 1 : number * Calculate(number - 1);
+        }
+
```

Comment:
Method names should follow PascalCase naming convention in C#. Please rename the `calculate` method to `Calculate`.